app/src/org/commcare/utils/SigningUtil.java

```diff
@@ -33,9 +37,14 @@
  * And we can then verify that the profile link was in fact signed (using SHA256withRSA) by
  * the CommCareHQ private key
  *
- * Created by wpride1 on 9/11/15.
+ * @author Will Pride (wpride@dimagi.com)
  */
 public class SigningUtil {
+    private final static ArrayList<Pattern> WHITELISTED_URL_HOSTS_REGEX = new ArrayList<>();
+
+    static {
+        WHITELISTED_URL_HOSTS_REGEX.add(Pattern.compile("^*.commcarehq.org"));
```

Comment:
`^*` isn't valid, should be `"^.*.commcarehq.org"` or could use `".commcarehq.org$"` (or `"\.commcarehq\.org$"` to be explicit that you actually want periods)